src/main/kotlin/com/bardsoftware/papeeria/backend/cosmas/CosmasGoogleCloudService.kt

```diff
@@ -226,6 +226,59 @@ class CosmasGoogleCloudService(private val bucketName: String,
         responseObserver.onCompleted()
     }
 
+    override fun deleteFile(request: CosmasProto.DeleteFileRequest, responseObserver: StreamObserver<CosmasProto.DeleteFileResponse>) {
+        LOG.info("Get request for delete file \"${request.fileName}\" # ${request.fileId}")
```

Comment:
You can use [triple-quoted string](https://kotlinlang.org/docs/reference/basic-types.html#string-literals) to get rid of escaping.

Reply:
done